.github/workflows/docs-ci.yml

```diff
@@ -6,7 +6,40 @@ on:
   push:
     branches: [ 'main', 'release/v*' ]
 
+env:
+  version-path: ${{ startsWith(github.ref, 'refs/heads/release/v') && github.ref_name || 'next' }}
+
 jobs:
+  symlink:
+    runs-on: ubuntu-22.04
+    concurrency:
+      group: symlink-${{ github.workflow }}-${{ github.ref }}
+      # We don't want to cancel in-progress jobs against main because that might leave the upload in a bad state.
+      cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Make Version Symlinks
+        if: ${{ startsWith(env.version-path, 'release/v') && github.event_name == 'push' }}
```

Comment:
I don't understand the "weird" part.  Variables are part of Workflows.  Are they unstable?  As for why I did it this way... as mentioned above, it's just easier to test.

Reply:
The "weird" part is that you are attaching specific semantics to a directory path name that could otherwise be any string - and to understand the `if` statement, now I have to go understand what `version-path` is. Additionally, if we are going to change `version-path` in the future, we know need to audit everything that uses it to understand what `version-path` _actually_ is. In this PR, `version-path` *isn't* just a directory name, but it is also being used for this dual-purpose of gating whether a step runs or not, and that's why it's "weird".